### ISSUE TITLE
npm run build failure

### ISSUE THREAD

**Issue by nterray** [Microsoft/accessibility-insights-web#363]
## Describe the bug

When I run `npm run build`, the build is failing with "Module not found" error. Both on master and v2.0.1.

**To Reproduce**

1. git clone
2. npm install
3. npm run build

## Expected behavior

The build should succeed.

## Screenshots

```
> accessibility-insights-web@1.0.0 build /home/nicolas/accessibility-insights-web
> grunt

Running "clean:intermediates" (clean) task
>> 1 path cleaned.

Running "exec:webpack-dev" (exec) task
Starting type checking service...
Using 1 worker with 2048MB memory limit
Hash: 151b0c4be3a499a47353
Version: webpack 4.29.1
Time: 19620ms
Built at: 03/14/2019 11:01:28 AM
                    Asset      Size       Chunks             Chunk Names
     background.bundle.js  5.48 MiB   background  [emitted]  background
 background.bundle.js.map   5.5 MiB   background  [emitted]  background
    detailsView.bundle.js  7.69 MiB  detailsView  [emitted]  detailsView
detailsView.bundle.js.map  7.29 MiB  detailsView  [emitted]  detailsView
       devtools.bundle.js  4.01 KiB     devtools  [emitted]  devtools
   devtools.bundle.js.map  3.52 KiB     devtools  [emitted]  devtools
       injected.bundle.js   7.3 MiB     injected  [emitted]  injected
   injected.bundle.js.map  6.93 MiB     injected  [emitted]  injected
       insights.bundle.js  6.43 MiB     insights  [emitted]  insights
   insights.bundle.js.map  6.04 MiB     insights  [emitted]  insights
          popup.bundle.js  7.12 MiB        popup  [emitted]  popup
      popup.bundle.js.map  6.69 MiB        popup  [emitted]  popup
Entrypoint injected = injected.bundle.js injected.bundle.js.map
Entrypoint popup = popup.bundle.js popup.bundle.js.map
Entrypoint insights = insights.bundle.js insights.bundle.js.map
Entrypoint detailsView = detailsView.bundle.js detailsView.bundle.js.map
Entrypoint devtools = devtools.bundle.js devtools.bundle.js.map
Entrypoint background = background.bundle.js background.bundle.js.map
[0] multi ./src/injected/stylesheet-init.ts ./src/injected/client-init.ts 40 bytes {injected} [built]
[1] multi ./src/views/insights/initializer.ts 28 bytes {insights} [built]
[2] multi ./src/DetailsView/details-view-initializer.ts 28 bytes {detailsView} [built]
[3] multi ./src/devtools/dev-tool-init.ts 28 bytes {devtools} [built]
[4] multi ./src/background/background-init.ts 28 bytes {background} [built]
[./src/DetailsView/details-view-initializer.ts] 15.8 KiB {detailsView} [built]
[./src/background/background-init.ts] 5.07 KiB {background} [built]
[./src/background/browser-adapter.ts] 7.02 KiB {popup} {insights} {detailsView} {background} [built]
[./src/common/fabric-icons.ts] 2.32 KiB {injected} {popup} {insights} {detailsView} [built]
[./src/common/url-parser.ts] 798 bytes {popup} {detailsView} [built]
[./src/common/url-validator.ts] 3.99 KiB {popup} {background} [built]
[./src/injected/client-init.ts] 826 bytes {injected} [built]
[./src/injected/stylesheet-init.ts] 306 bytes {injected} [built]
[./src/popup/scripts/popup-init.ts] 963 bytes {popup} [built]
[./src/popup/scripts/popup-initializer.ts] 10.3 KiB {popup} [built]
    + 1469 hidden modules

ERROR in multi ./src/devtools/dev-tool-init.ts
Module not found: Error: Can't resolve '/home/nicolas/accessibility-insights-web/src/devtools/dev-tool-init.ts' in '/home/nicolas/accessibility-insights-web'
 @ multi ./src/devtools/dev-tool-init.ts devtools[0]

ERROR in /home/nicolas/accessibility-insights-web/src/injected/main-window-context.ts
ERROR in /home/nicolas/accessibility-insights-web/src/injected/main-window-context.ts(3,22):
TS6053: File '/home/nicolas/accessibility-insights-web/src/injected/Window.d.ts' not found.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts(7,65):
TS2307: Cannot find module '../Stubs/chrome-adapter-stub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts(9,14):
TS2420: Class 'PortWithTabTabIdStub' incorrectly implements interface 'PortWithTabId'.
  Property 'postMessage' is missing in type 'PortWithTabTabIdStub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts(12,14):
TS2339: Property 'name' does not exist on type 'PortWithTabTabIdStub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts(13,14):
TS2339: Property 'onMessage' does not exist on type 'PortWithTabTabIdStub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/chrome-adapter-mock.ts(14,14):
TS2339: Property 'onDisconnect' does not exist on type 'PortWithTabTabIdStub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/port-on-disconnect-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/port-on-disconnect-mock.ts(6,36):
TS2307: Cannot find module '../Stubs/chrome-adapter-stub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/port-on-message-mock.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/mock-helpers/port-on-message-mock.ts(6,35):
TS2307: Cannot find module '../Stubs/chrome-adapter-stub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/tests/background/dev-tools-listener.test.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/tests/background/dev-tools-listener.test.ts(14,26):
TS2307: Cannot find module '../../Stubs/chrome-adapter-stub'.

ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/tests/background/dev-tools-listener.test.ts
ERROR in /home/nicolas/accessibility-insights-web/src/tests/unit/tests/background/dev-tools-listener.test.ts(41,27):
TS2339: Property 'name' does not exist on type 'PortWithTabTabIdStub'.
>> Exited with code: 2.
>> Error executing child process: Error: Process exited with code 2.
Warning: Task "exec:webpack-dev" failed. Use --force to continue.

Aborted due to warnings.
```

## Extension (please complete the following information)

- OS: Linux Mint
- Version: 19.1
- npm: 6.9.0

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? It depends :)

## Did you search for similar existing issues?

Did you search for similar issues in our github issues or on [stackoverflow](https://stackoverflow.com/questions/tagged/accessibility-insights)? Yes

**Comment by smoralesd** [comment]
Hi @nterray 
Thanks for you interest on Accessibility Insights for Web.
Can you confirm which version of node you're using?
We'll run some preliminary investigation be triaging this issue.

**Comment by Shobhit1** [comment]
Hi @nterray 

Thanks for reporting the issue. We found an issue which was causing this and should be fixed with #366 .
Can you try updating your fork and re run `npm run build`? 
Let me know if it works this time.
Thank you.

**Comment by nterray** [comment]
Yeah, it works!

Thanks
